deadlight/www.ubuntu.com
# Bootstack service description

## Done

Updated Bootstack service description

## QA

Go to: /legal/bootstack/service-description
See that the end of section 2.7 matches the copy given in #766

## Issue / Card

Fixes: #766
